client-app/shared/static-content/components/slider.vue

```diff
@@ -6,7 +6,7 @@
       <Swiper :slides-per-view="1" class="w-full" :modules="modules" :navigation="navigationOptions">
         <SwiperSlide v-for="(item, index) in model.slides" :key="index" class="text-center">
           <div>
-            <VcImage :src="item.image" class="place-items-center" lazy />
+            <VcImage :src="item.image" class="place-items-center" :lazy="index > 0" />
```

Comment:
rename index to blockOrder. 
it should be first

Reply:
Why? It's already used everywhere and is default https://vuejs.org/api/built-in-directives.html#v-for